apps-rendering/src/components/metadata.tsx

```diff
@@ -34,12 +46,100 @@ const withBylineTextStyles = css`
 	padding-top: ${remSpace[1]};
 `;
 
+const blogStyles = css`
+	display: block;
+	margin-bottom: ${remSpace[2]};
+	${from.desktop} {
+		width: ${wideColumnWidth}px;
+	}
+`;
+
+const metaBottomStyles = css`
+	display: flex;
+	border-top: 1px solid rgba(255, 255, 255, 0.4);
+
+	${from.desktop} {
+		border-top: 1px solid ${neutral[86]};
+	}
+`;
+
+const toggleStyles = css`
+	flex-grow: 1;
+	padding-top: 8px;
+`;
+
+const liveblogSidePadding = css`
+	${sidePadding}
+
+	padding-bottom: ${remSpace[3]};
+
+	${liveblogPhabletSidePadding}
+
+	${from.desktop} {
+		padding-left: 0;
+		padding-right: 0;
+	}
+`;
+
+const toggleOverrideStyles = css`
```

Comment:
Great point. Have made a PR in source to address this https://github.com/guardian/source/pull/1134

Reply:
new version of toggle switch has been used. Most of the styling that could get moved to source are now removed from metadata file. But remember the colours for dark background (which is our designs for breakpoints below desktop) are still in AR, because the agreement for source toggle switch was to have a default colouring only on a light background.